src/kaltura-player.ts

```diff
@@ -823,6 +824,14 @@ export class KalturaPlayer extends FakeEventTarget {
     return this._localPlayer.Error;
   }
 
+  private _configureMediaMetadata(mediaConfig: KPMediaConfig): void {
```

Comment:
if it's relevant only for mobile maybe it better to call it something like updateSessionForMobile() it would be more descriptive (the fact that it is configured media metadata can already be seen in the function declaration)

Reply:
@JonathanTGold the requirement at the moment is for mobile but it is not limited to it. could also be TVs, tablets, etc..

Reply:
got it bot anyway this is too generic name (and confusing with our media meta data object) it would be better to find a more specific name